### PR TITLE
06 time management prog

### PR DESCRIPTION
**Pull request by DomninDmitry-dev**
Homework TimeManagement test programm.
This test programm read attribute file "/sys/class/my_time_mod/absolute-time" and print time of kernel-space and user-space.

Build this programm:
make SYSTEM=qemu
Clean this module:
make clean

### CODE REVIEW

File: 06-TimeManagement/user_space/myprog.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	if(ret < 0) {
+                fprintf(stderr, "Read error %s\n", INPUT_ATTR);
+                exit(EXIT_FAILURE);
+        }
+
+	ret = gettimeofday(&t, NULL);
+	if (ret < 0)
+		exit(EXIT_FAILURE);
+
+	printf("User-space get absolute time: %ld.%ld\n", t.tv_sec, t.tv_usec);
```
Comment (yekovalyov) [inline 06-TimeManagement/user_space/myprog.c]: please run your program at least 20 times and check usec value. Analyze obtained values.

**Comment by yekovalyov** [comment]
Please check your code using checkpatch.pl

**Comment by DomninDmitry-dev** [comment]
Done. Add log file.

File: 06-TimeManagement/user_space/log.txt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+User-space get absolute time: 1573486487.784704
+Kernel-space get absolute time: 1573486487.784672
+
+$ ./myprog 
+User-space get absolute time: 1573486491.688454
+Kernel-space get absolute time: 1573486491.688420
+
+$ ./myprog 
+User-space get absolute time: 1573486494.69452
+Kernel-space get absolute time: 1573486494.69421
```
Comment (yekovalyov) [inline 06-TimeManagement/user_space/log.txt]: is it 1573486494 sec  69421 microseconds or
1573486494 sec 694210 microseconds?

**Comment by DomninDmitry-dev** [comment]
Done

**Pull request merged**